api/v1beta1/types.go

```diff
@@ -358,14 +358,15 @@ type AddressPair struct {
 }
 
 type BastionStatus struct {
-	ID         string              `json:"id,omitempty"`
```

Comment:
Should probably put bastion ID and State back, too.

Reply:
It was the idea that State and ID goes into Server to mimic what's on OpenStackMachineStatus. No strong opinion from my side here.